### DISCUSSION TITLE
The apk size created in flet 0.25.2 version is much larger than previous versions.

### DISCUSSION THREAD

**Discussion by yucel685** [flet-dev/flet#5920] [General]
### Duplicate Check

- [X] I have searched the [opened issues](https://github.com/flet-dev/flet/issues) and there are no duplicates

### Describe the bug

The apk size created in flet 0.25.2 version is much larger than previous versions. Application opening time on Android is longer than the old version

### Code sample

kodda sorun yok.

### To reproduce

hata raporu yok

### Screenshots / Videos

<details open>
<summary>Captures</summary>

[Upload media here]

</details>

### Operating System

Windows

### Operating system details

intel core i5

### Flet version

0.25.2

### Regression

Yes, it used to work in a previous Flet version (please specify the version in additional details)

### Logs

<details open><summary>Logs</summary>

```console
[Paste your logs here]
```

</details>

**Comment by HappyCoder256** [comment]
Make sure there are no extra files in your project root directory if the .env file is there. Locate it somewhere else. 

Ensure to treeshake all unnecessary and unused libraries in your project. 

Add also after the update, Some libraries where included and have to be bundled with the Apk. 

But first check in ur root folder for unnecessary files
